oom_kill/manifest.json

```
@@ -0,0 +1,28 @@
+{
+  "display_name": "OOM Kill",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "oom_kill",
+  "metric_prefix": "oom_kill.",
+  "metric_to_check": "oom_kill.oom_process.count",
+  "creates_events": true,
+  "short_description": "Track process OOM kills by the system or cgroup.",
+  "guid": "4b8e9c18-1a13-43b0-a03c-186eb3221147",
+  "support": "core",
+  "supported_os": [
+    "linux",
```

```suggestion
    "linux"
```

This was causing CI to fail because trailing `,` is not valid in json